VMEncryption/main/oscrypto/centos_68/encryptstates/SplitRootPartitionState.py

```diff
@@ -192,7 +192,9 @@ def _size_to_sectors(self, bytes_, unit, sector_size):
             "YiB":  1024**8  # yobibyte
         }
 
-        if unit not in exponents.keys():
+        #P2 - if unit not in exponents.keys():
```

Comment:
comment: python3 returns a lazy iterable rather than a full list

Reply:
`# use list() for python2 + python3 compat and to avoid lazy iterable evaluation`